I'm sorry, but an uncaught exception occurred.

Compiling ATL code at game/scenes.rpy:79
  File "game/day4.rpy", line 181, in script
    "???" "{font=fonts/NewAster.ttf} {cps=*0.2} MY... BO... CO... HE... CLO...{/cps}{/font}"
  File "renpy/common/00library.rpy", line 295, in say
    who(what, interact=interact, *args, **kwargs)
NameError: name 'hbox' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/day4.rpy", line 181, in script
    "???" "{font=fonts/NewAster.ttf} {cps=*0.2} MY... BO... CO... HE... CLO...{/cps}{/font}"
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\ast.py", line 694, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\exports.py", line 1320, in say
    renpy.store.say(who, what, *args, **kwargs)
  File "renpy/common/00library.rpy", line 295, in say
    who(what, interact=interact, *args, **kwargs)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\character.py", line 1127, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\character.py", line 842, in do_display
    **display_args)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\character.py", line 591, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\ui.py", line 289, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 2690, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 3074, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 534, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 534, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 534, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 534, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 534, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 534, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 534, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\display\core.py", line 524, in visit_all
    for d in self.visit():
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\atl.py", line 577, in visit
    block = self.compile()
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\atl.py", line 484, in compile
    block = self.atl.compile(self.context)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\atl.py", line 679, in compile
    statements = [ i.compile(ctx) for i in self.statements ]
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\atl.py", line 892, in compile
    child = ctx.eval(expr)
  File "C:\Users\satel\Downloads\renpy-6.99.12.2-sdk\renpy\atl.py", line 246, in eval
    return eval(expr, renpy.store.__dict__, self.context)  # @UndefinedVariable
  File "<string>", line 1, in <module>
NameError: name 'hbox' is not defined

Windows-8-6.2.9200
Ren'Py 7.2.2.491
UserInterface 0.0
Sat May 18 15:22:35 2019
